tools/gyp/tools/pretty_gyp.py

```diff
@@ -118,23 +120,24 @@ def prettyprint_input(lines):
   basic_offset = 2
   last_line = ""
   for line in lines:
-    line = line.strip('\r\n\t ')  # Otherwise doesn't strip \r on Unix.
-    if len(line) > 0:
-      brace_diff = 0
-      if not COMMENT_RE.match(line):
+    if COMMENT_RE.match(line):
+      print(line)
```

Comment:
Does this not change the behavior? Before, it used to strip '\r\n\t ' from the line?

Reply:
It looks like this is still doing that at [126](https://github.com/nodejs/node/pull/28573/files#diff-d8169636d16b79452ea71e3d0e9a9b7bR126) - This just now prints the line if it matches the regex, otherwise strips - Unlike before where it stripped it if failed the regex immediately.